tests/Unit/IsPhoneTest.php

```diff
@@ -0,0 +1,53 @@
+<?php declare(strict_types = 1);
+
+namespace PdTests\Unit;
+
+require __DIR__ . '/../bootstrap.php';
+
+/**
+ * @testCase
+ */
+final class IsPhoneTest extends \Tester\TestCase
+{
+	/**
+	 * @dataProvider provideValidInput
+	 */
+	public function testValid(string $input): void
+	{
+		\Tester\Assert::true(\Pd\Utils\Validators::isPhone($input));
+	}
+
+
+	/**
+	 * @dataProvider provideInvalidInput
+	 */
+	public function testInvalid(string $input): void
+	{
+		\Tester\Assert::false(\Pd\Utils\Validators::isPhone($input));
+	}
+
+
+	public function provideValidInput(): array
+	{
+		return [
+			['+420 543 236 506'],
```

Comment:
Na projektech máme i Maďarské jazykové mutace, tam začíná telefonní číslo na +36 (a obecně je to od jedničky), tak by to chtělo na tuto eventualitu myslet taky.

Reply:
https://github.com/peckadesign/Gant2017/blob/master/app/UserModule/FrontModule/Form/PhoneInputModifier.php#L9

Reply:
dobrá připomínka, ale imho mimo scope zadání úkolu, znamenalo by to doplnit pattern a upravit metodu `Validators::isPhone()`, můžu udělat v novém PR, ok?

Reply:
Jasně připrav následně do vlastního PR.

Reply:
https://github.com/peckadesign/utils/pull/4